site-src/guides/api-overview.md

```diff
@@ -0,0 +1,129 @@
+
+## Roles and personas
+
+In this documentation we refer to three primary personas:
+
+- Application Developer
+- Namespace Administrator
+- Cluster Administrator
+
+## Resource Model 
+
+There are two main objects in the AdminNetworkPolicy API resource model
+
+- **AdminNetworkPolicy (ANP)**
+
+- **BaselineAdminNetworkPolicy (BANP)**
+
+## General Notes
+
+- Any well defined AdminNetworkPolicy rules should
+be read as-is, i.e. there will not be any implicit isolation effects for the Pods
+selected by the AdminNetworkPolicy, as opposed to what NetworkPolicy rules imply.
+
+- As of `v1alpha1` of this API we focus primarily on E/W cluster traffic and 
+do not address N/S (Ingress/Egress) use cases. However this is an issue the community 
+would like to keep thinking about during further iterations, and a tracking issue 
+can be found/ commented on here ---> [issue #28](https://github.com/kubernetes-sigs/network-policy-api/issues/28)
+
+## The AdminNetworkPolicy Resource 
+
+The AdminNetworkPolicy (ANP) resource will help administrators set strict security 
+rules for the cluster, i.e. a developer CANNOT override these rules by creating 
+NetworkPolicies that apply to the same workloads as the AdminNetworkPolicy.
+
+### AdminNetworkPolicy Actions 
+
+Unlike the NetworkPolicy resource in which each rule represents an allowed
+traffic, AdminNetworkPolicies will enable administrators to set `Pass`,
+`Deny` or `Allow` as the action of each rule. AdminNetworkPolicy rules should
+be read as-is, i.e. there will not be any implicit isolation effects for the Pods
+selected by the AdminNetworkPolicy, as opposed to implicit deny NetworkPolicy rules imply.
+
+- **Pass**: Traffic that matches a `Pass` rule will skip all further rules from all
+  numbered ANPs and instead be enforced by the K8s NetworkPolicies.
+  If there is no K8s NetworkPolicy of BaselineAdminNetworkPolicy rule match
+  match, traffic will be governed by the implementation. For most implementations, 
+  this means "allow", but there may be implementations which have their own policies 
+  outside of the standard Kubernetes APIs.
+- **Deny**: Traffic that matches a `Deny` rule will be dropped.
+- **Allow**: Traffic that matches an `Allow` rule will be allowed.
+
+AdminNetworkPolicy `Deny` rules are useful for administrators to explicitly
+block traffic with malicious in-cluster clients, or workloads that pose security risks.
+Those traffic restrictions can only be lifted once the `Deny` rules are deleted,
+modified by the admin, or overridden by a higher priority rule.
+
+On the other hand, the `Allow` rules can be used to call out traffic in the cluster
+that needs to be allowed for certain components to work as expected (egress to
+CoreDNS for example). Those traffic should not be blocked when developers apply
+NetworkPolicy to their Namespaces which isolates the workloads.
+
+AdminNetworkPolicy `Pass` rules allows an admin to delegate security posture for
```

Comment:
what does security posture mean here?

Reply:
I guess you can think of this as terminal action (i.e allow or drop) It just allows the developer, with network policies) to decide what happens to the traffic